### PR TITLE
Uses cases update

### PR DESCRIPTION
**Pull request by fabriguespe**
Uses cases update with groups and frames use cases

### CODE REVIEW

File: docs/use-cases/commerce.mdx [hunk 1]
```diff
@@ -23,29 +23,23 @@ E-commerce platforms are constantly seeking ways to improve user experience, tru
 
 #### Inspiration
 
-- **Order confirmations**  
-   XMTP can be used to securely confirm orders, ensuring that both buyers and sellers have verifiable proof of purchase and sale without exposing sensitive information.
+- **Order confirmations**:XMTP can be used to securely confirm orders, ensuring that both buyers and sellers have verifiable proof of purchase and sale without exposing sensitive information.
```
Comment (scotboyd) [inline docs/use-cases/commerce.mdx:26]:
```suggestion
- **Order confirmations**: XMTP can be used to securely confirm orders, ensuring that both buyers and sellers have verifiable proof of purchase and sale without exposing sensitive information.
```

File: docs/use-cases/commerce.mdx [hunk 2]
```diff
@@ -23,29 +23,23 @@ E-commerce platforms are constantly seeking ways to improve user experience, tru
...
-   For more complex financial dealings within e-commerce, such as layaway plans or financing offers, XMTP provides a secure channel for handling these transactions.
+- **Finance transactions**: For more complex financial dealings within e-commerce, such as layaway plans or financing offers, XMTP provides a secure channel for handling these transactions.
 
-- **Escrows**  
-   Escrow services can be handled securely through XMTP, assuring both parties of the integrity of the transaction.
+- **Escrows**: Escrow services can be handled securely through XMTP, assuring both parties of the integrity of the transaction.
 
-- **Approval requests**  
-   Whether it's for returns, custom orders, or any situation requiring explicit approval, XMTP facilitates these requests securely.
+- **Approval requests** : Whether it's for returns, custom orders, or any situation requiring explicit approval, XMTP facilitates these requests securely.
```
Comment (scotboyd) [inline docs/use-cases/commerce.mdx:40]:
```suggestion
- **Approval requests**: Whether it's for returns, custom orders, or any situation requiring explicit approval, XMTP facilitates these requests securely.
```

File: docs/use-cases/dao.mdx [hunk 3]
```diff
@@ -18,14 +18,13 @@ import snapshot from "./img/snapshot.jpg";
 
 #### Inspiration
 
-- **Vote notifications**  
-  Implementing a system for sending secure and verifiable notifications related to voting within the DAO. This can ensure that all members are informed of critical decisions and that their participation is accurately recorded.
+- **Vote notifications**: Implementing a system for sending secure and verifiable notifications related to voting within the DAO. This can ensure that all members are informed of critical decisions and that their participation is accurately recorded.
 
-- **Subscription to a user**  
-  Enabling users to subscribe to specific members or activities within the DAO. This feature allows for targeted communication and keeps members engaged and informed about relevant updates.
+- **Subscription to a user**:Enabling users to subscribe to specific members or activities within the DAO. This feature allows for targeted communication and keeps members engaged and informed about relevant updates.
```
Comment (scotboyd) [inline docs/use-cases/dao.mdx:23]:
```suggestion
- **Subscription to a user**: Enabling users to subscribe to specific members or activities within the DAO. This feature allows for targeted communication and keeps members engaged and informed about relevant updates.
```

File: docs/use-cases/dao.mdx [hunk 4]
```diff
@@ -18,14 +18,13 @@ import snapshot from "./img/snapshot.jpg";
...
 
-- **Group chat between members**  
-  Creating a secure and transparent platform for members to communicate with each other. Such a platform could be built with features to authenticate messages and participants, thus creating a more trustworthy environment.
+- **Group chat between members** : Creating a secure and transparent platform for members to communicate with each other. Such a platform could be built with features to authenticate messages and participants, thus creating a more trustworthy environment.
```
Comment (scotboyd) [inline docs/use-cases/dao.mdx:25]:
```suggestion
- **Group chat between members**: Creating a secure and transparent platform for members to communicate with each other. Such a platform could be built with features to authenticate messages and participants, thus creating a more trustworthy environment.
```

**Review by scotboyd — CHANGES_REQUESTED** [review]

**Review by fabriguespe — COMMENTED** [review]
Thanks for the changes!

**Pull request merged**